src/data/dfinsupp.lean

```diff
@@ -452,6 +452,48 @@ by simp
 lemma erase_ne {i i' : ι} {f : Π₀ i, β i} (h : i' ≠ i) : (f.erase i) i' = f i' :=
 by simp [h]
 
+section update
+
+variables (f : Π₀ i, β i) (i : ι) (b : β i) [decidable (b = 0)]
+
+/-- Replace the value of a `Π₀ i, β i` at a given point `i : ι` by a given value `b : β i`.
+If `b = 0`, this amounts to removing `i` from the support.
+Otherwise, if `i` was not in the support, it is added to it.  -/
```

Comment:
```suggestion
Otherwise, if `i` was not in the support, it is added to it.

This is the (dependent) finitely-supported version of `function.update`. -/
```